gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/setConfig.ts

```diff
@@ -69,21 +68,48 @@ export default class SetConfig extends SolanaCommand {
 
     logger.log('Config information:', input)
     await prompt(`Continue setting config on ${state.toString()}?`)
```

Comment:
This prompt is in many commands. It shouldn't be at tx creation time, but on execution. Please change it in every command that appears like that

Reply:
Changing them. The only drawback is that users will not be prompted while using multisig, that's why I had left them.